frontend/src/components/Themes/Themes.css

```diff
@@ -0,0 +1,8 @@
+.w-70 {
```

Comment:
I think tachyons already has this classes defined. Try to remove this classes and test it please